http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/NodepoolComponent.scala

```diff
@@ -131,6 +131,21 @@ object nodepoolQuery extends TableQuery(new NodepoolTable(_)) {
       claimedNodepoolOpt <- unclaimedNodepoolOpt.flatTraverse(n => getMinimalById(n.id))
     } yield claimedNodepoolOpt
 
+  def claimNodepoolWithConfig(clusterId: KubernetesClusterLeoId,
```

Comment:
An aside: @jdcanas discussed in our 1-1 removing the "claim nodepool" functionality. It's not used in production and we've never actually used it in a workshop. So it really just adds complexity with no real benefit.

Reply:
created a [ticket](https://broadworkbench.atlassian.net/browse/IA-2644)